api/app/signals/apps/feedback/models.py

```diff
@@ -0,0 +1,51 @@
+import secrets
+import uuid
+from datetime import timedelta
+
+from django.contrib.gis.db import models
+from django.utils import timezone
+
+from signals.apps.feedback.managers import FeedbackManager
+from signals.apps.signals.models import Signal
+
+FEEDBACK_EXPECTED_WITHIN_N_DAYS = 14  # move to general settings file
+
+
+def generate_token():
```

Comment:
Do we really need this function? See comment in the Feedback model

Reply:
see previous comment